testsuite/model/src/test/java/org/keycloak/testsuite/model/ClientModelTest.java

```diff
@@ -122,10 +122,8 @@ public void testClientsBasics() {
             });
 
             withRealm(realmId, (session, realm) -> {
-                ClientModel client = session.clients().searchClientsByClientIdStream(realm, "%", 0, 10).findFirst().orElse(null);
-                assertThat(client, notNullValue());
-                assertThat(client.getId(), is(equalTo(searchClient.getId())));
-                assertThat(client.getClientId(), is(equalTo(searchClientId)));
+                // when searching by "%" all entries are expected
+                assertThat(session.clients().searchClientsByClientIdStream(realm, "%", 0, 10).count(), is(equalTo(2L)));
```

Comment:
Just out of curiosity: is this change required for this PR?

Reply:
Without it model tests fails, because it's not guaranteed which of the two clients is returned. 

As we are skipping the profile in GHA it won't be visible there, but locally it fails. We can move it to another issue if it makes more sense.

Reply:
I thought you'd say that, just wanted to make sure that was really the case ;) LGTM